openshift/templates/api/api-deploy.yaml

```diff
@@ -265,6 +265,11 @@ parameters:
       the server instance.
     required: true
     value: identity-kit-controller
+  - name: DB_APP_NAME
+    displayName: App Name
+    description: Used to group components together in the OpenShift console.
+    required: true
+    value: issuer-kit
```

Comment:
Similar/related to the param files: please have this param besides the `DB_SERVICE` param for ease of maintenence.

Reply:
moved params around and added DB_SUFFIX